ebml_matroska.xml

```diff
@@ -404,7 +404,7 @@
     </restriction>
   </element>
   <element name="ColourSpace" path="0*1(\Segment\Tracks\TrackEntry\Video\ColourSpace)" cppname="VideoColourSpace" id="0x2EB524" type="binary" maxOccurs="1" minver="1" webm="0" size="4">
-    <documentation lang="en">Same value as in AVI (32 bits).</documentation>
+    <documentation lang="en">Specify the pixel format used for the Track's data as a fourcc. This value is similar in scope to the biCompression value of AVI's BITMAPINFOHEADER. This Element is MANDATORY in TrackEntry when the CodecID Element of the TrackEntry is set to "V_Uncompressed".</documentation>
```

Comment:
`FourCC` is actually an abbreviation for [four-character code](https://en.wikipedia.org/wiki/FourCC). Therefore `fourcc` should be `FourCC`.